package.json

```diff
@@ -6,7 +6,8 @@
   "bin": "./index.js",
   "scripts": {
     "start": "node .",
-    "test": "npm-run-all -s lint cover check-coverage",
+    "nonsense": "LC_ALL=C sed -n 1l index.js",
```

Comment:
What's going on here?

Reply:
This commit is called 'DELETE ME: not a commit '. Does it need to be rebased out?

Reply:
_Don't Panic_

Reply:
That commit was just to test something on CI, but has been scrubbed!